### PR TITLE
Remove Employments dummy endpoint

### PR DESCRIPTION
**Pull request by dabd**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -62,109 +62,102 @@ application.router=prod.Routes
...
+  }
+
+  uk.gov.hmrc.selfassessmentapi.controllers.definition.DocumentationController = {
+    needsHeaderValidation = false
+    needsAuth = false
+    needsLogging = false
+    needsAuditing = false
+  }
+
+  uk.gov.hmrc.selfassessmentapi.controllers.sandbox.EmploymentsController = {
```
Comment (danmassie) [inline conf/application.conf]: This should be gone

File: it/uk/gov/hmrc/selfassessmentapi/AuthorisationSpec.scala [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 
   "if the user is not authorised for the resource they" should {
     "receive 401" in {
-      given().userIsNotAuthorisedForTheResource(saUtr)
+      given()
+        .userIsNotAuthorisedForTheResource(saUtr)
         .when()
-        .get(s"/$saUtr/employments")
-        .thenAssertThat().statusIs(401)
+        .get(s"/sandbox/$saUtr/self-employments/$selfEmploymentId")
```
Comment (danmassie) [inline it/uk/gov/hmrc/selfassessmentapi/AuthorisationSpec.scala]: No auth is applied to sandbox endpoints.

File: conf/application.conf [hunk 3]
```diff
@@ -62,109 +62,102 @@ application.router=prod.Routes
...
-    }
-
-    uk.gov.hmrc.selfassessmentapi.controllers.sandbox.SelfEmploymentsController = {
-      needsAuth = false
-      needsLogging = false
-      needsAuditing = false
+  }
+
+  uk.gov.hmrc.selfassessmentapi.controllers.sandbox.SelfEmploymentsController = {
+    needsAuth = true
```
Comment (danmassie) [inline conf/application.conf]: no auth

File: it/uk/gov/hmrc/selfassessmentapi/sandbox/SelfEmploymentControllerSpec.scala [hunk 4]
```diff
@@ -16,6 +16,7 @@ class SelfEmploymentControllerSpec extends BaseFunctionalSpec {
 
     "return a 201 response with links to newly created self employment" in {
       given()
+        .userIsAuthorisedForTheResource(saUtr)
```
Comment (danmassie) [inline it/uk/gov/hmrc/selfassessmentapi/sandbox/SelfEmploymentControllerSpec.scala]: no auth on sandbox

**Comment by danmassie** [comment]
@dabd tests fail on travis

**Comment by saurabharora80** [comment]
👍

**Pull request merged**